app/controllers/single_reservations_controller.rb

```diff
@@ -0,0 +1,84 @@
+class SingleReservationsController < ApplicationController
+
+  customer_tab  :all
+  before_action :authenticate_user!
+  load_resource :facility, find_by: :url_name
+  load_resource :instrument, through: :facility, find_by: :url_name
+  before_action :build_order
+  before_action { @submit_action = facility_instrument_single_reservations_path }
+
+  def new
+    options = current_user.can_override_restrictions?(@instrument) ? {} : { user: acting_user }
+    next_available = @instrument.next_available_reservation(after: 1.minute.from_now, duration: default_reservation_mins.minutes, options: options)
+    @reservation = next_available || default_reservation
+    @reservation.order_detail = @order_detail
+
+    authorize! :new, @reservation
+
+    @reservation.round_reservation_times
+    unless @instrument.can_be_used_by?(acting_user)
+      flash[:notice] = text(".acting_as_not_on_approval_list")
+    end
+    set_windows
+    render "reservations/new"
+  end
+
+  def create
```

Comment:
I copied this from `ReservationsController` and cleaned out some of the unneeded pieces (like merge orders). I still want to do some further refactors to remove some of the duplication.